### ISSUE TITLE
Error in compacting object

### ISSUE THREAD

**Issue by tristanleboss** [mck89/peast#13]
The compacter will output such code:

```
{
	'tab': 'active.tab',
	'sidepane',
}
```

when encoutering such JS code:

```
{
	'tab': 'active.tab',
	'sidepane': 'sidepane',
}
```

Unfortunately, this is not valid JS for latest Chrome:

![image](https://user-images.githubusercontent.com/1431508/66715381-f6b4e780-edc2-11e9-8dca-5440d76dc194.png)

**Comment by tristanleboss** [comment]
By the way, I use PEAST (and it's compact mode) for months on hundreds of JS files so I can tell you it's really stable!
The cool thing is that I can point you some interesting edge cases ;)

**Comment by mck89** [comment]
Glad to hear that it's working well, please keep reporting whenever you find bugs, you are very helpful 👍 .

There is a missing check for the cases where this syntax is valid becuase:
`a = {b}`
is valid, but:
`a = {"b"}`
is invalid.

I will work to fix this soon.

**Comment by mck89** [comment]
Fixed in the new version 1.9.4
